common/src/main/java/me/shedaniel/architectury/registry/trade/TradeRegistry.java

```diff
@@ -59,4 +144,38 @@ public static void registerTradeForWanderingTrader(boolean rare, VillagerTrades.
         throw new AssertionError();
     }
     
+    /**
+     * Override the max possible offers the wandering trader can have. This does not affect the rare trade.
+     * @param maxOffers  Max possible offers a villager can have.
+     */
+    public static void setWanderingTraderMaxOffers(int maxOffers) {
+        if (maxOffers < 0) {
+            throw new IllegalArgumentException("Wandering trader's max offers has to be at least 0!");
+        }
+        
+        WANDERING_TRADER_MAX_OFFER_OVERRIDE = maxOffers;
+    }
+    
+    /**
+     * @return Max offers for the wandering trader. Returning null means no override exists
+     */
+    public static Integer getWanderingTraderMaxOffers() {
```

Comment:
getWanderingTraderMaxOffersOverride or something along the lines of that but shorter

also @Nullable

Reply:
Without just shorten `WanderingTrader` into `WT` what would be a shorter version for it? I don't think in general longer method names are bad.

 added @nullable for `getVillagerMaxOffers` too

Reply:
tbh considering this again maybe it is best to not expose this method in the "api", using WT is kinda meh and I don't feel like that's where we want to simplify at all, would 100% want the full WanderingTrader instead